packages/@aws-cdk/assert/README.md

```diff
@@ -79,3 +79,26 @@ expect(stack).to(haveResource('AWS::CertificateManager::Certificate', {
     // Note: some properties omitted here
 }));
 ```
+### Check existence of an output
+`haveOutput` assertion can be used to check that a stack contains specific output. 
+Parameters to check against can be:
+- output name
+- output value
+- export name
+
+If output value is provided, at least one of \[output name, export name\] should be proviled as well
```

Comment:
```suggestion
If `outputValue` is provided, at least one of `outputName`, `exportName` should be provided as well
```

Reply:
Also - the code suggests either `outputName` or `exportName` is required **regardless** of whether `outputValue` is provided or not.